fractal_server/app/routes/auth.py

```diff
@@ -56,24 +62,18 @@
 )
 
 
-@router_auth.patch("/me", response_model=UserRead)
+@router_auth.patch("/current-user", response_model=UserRead)
```

Comment:
Let's start adding trailing slash to our custom endpoints, even if they would be added automatically at the end of this file. This will make things more readable, as one would immediately see the correct path.